app/scripts/modules/amazon/src/function/configure/FunctionDebugAndErrorHandling.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+
+import {
+  FormikFormField,
+  IWizardPageComponent,
+  HelpField,
+  TextInput,
+  ReactSelectInput,
+  ValidationMessage,
+} from '@spinnaker/core';
+import { FormikProps, FormikErrors } from 'formik';
+import { IAmazonFunctionUpsertCommand } from 'amazon/index';
+import { IAmazonFunction } from 'amazon/domain';
+
+export interface IFunctionDebugAndErrorHandlingProps {
+  formik: FormikProps<IAmazonFunctionUpsertCommand>;
+  isNew?: boolean;
+  functionDef: IAmazonFunction;
+}
+
+export interface IFunctionDebugAndErrorHandlingState {
+  some: string;
+}
+
+export class FunctionDebugAndErrorHandling
+  extends React.Component<IFunctionDebugAndErrorHandlingProps, IFunctionDebugAndErrorHandlingState>
+  implements IWizardPageComponent<IAmazonFunctionUpsertCommand> {
+  constructor(props: IFunctionDebugAndErrorHandlingProps) {
+    super(props);
+  }
+  public validate = (values: IAmazonFunctionUpsertCommand): FormikErrors<IAmazonFunctionUpsertCommand> => {
+    const errors = {} as any;
+    if (
+      values.deadLetterConfig.targetArn &&
+      !values.deadLetterConfig.targetArn.match(/arn:aws[a-zA-Z-]?:[a-zA-Z_0-9.-]+:./)
+    ) {
+      errors.deadLetterConfig = 'Invalid target ARN.';
+    }
+    return errors;
+  };
+
+  public componentDidMount() {
```

Comment:
This can be moved into the constructor, and you can just assign the initial state:
`this.state = { some: '' };`

Reply:
done. will appear in the next push.